framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -660,6 +635,52 @@ public String toString() {
         }
     }
 
+    private final class CheckerMessageComparator implements Comparator<CheckerMessage> {
```

Comment:
CI should have issued a warning due to the lack of documentation here.
I tracked it down to a bug in Javadoc (!), where it is skipping the class even if directed to process private elements.

Reply:
Done.